.github/ISSUE_TEMPLATE/filter.yml

```diff
@@ -0,0 +1,50 @@
+name: Filter Request
+title: "[FILTER]: "
+labels: filter request
+description: Report an issue with the filter
+body:
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Checklist
+      description: Confirm all of these are checked and valid.
+      options:
+        - label: I have read and agree to the README (found on https://github.com/Minehut/Meta)
+          required: true
+        - label: I have ensured the phrase is not properly filtered on the platform
+          required: true
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Which filter has the issue?
+      multiple: true
+      options:
+        - Discord
+        - Minecraft
+    validations:
+      required: true
+  - type: dropdown
+    id: filter
+    attributes:
+      label: Should the phrase be filtered or unfiltered?
+      options:
+        - Filter
```

Comment:
`Filtered` and `Unfiltered`
https://www.merriam-webster.com/dictionary/unfiltered unfiltered is one word here

Reply:
I'm confused, what do you want me to change :?

Reply:
I mean, ill captilize them, but idk if that's what you want.

Reply:
I want the options to be `Filtered` and `Unfiltered` specifically, the current options look odd